.gitignore

```diff
@@ -3,7 +3,12 @@ __pycache__
 .history
 .hypothesis/
 .tox/
-build/
```

Comment:
This is pretty unusual, will each deployment need to be added to the `.gitignore`? I guess this links with #18